plugins/webhook-event-forwarder/rollup.config.mjs

```diff
@@ -1,7 +1,35 @@
 import commonjs from '@rollup/plugin-commonjs'
 import json from '@rollup/plugin-json'
-import nodeResolve from '@rollup/plugin-node-resolve'
+import resolve from '@rollup/plugin-node-resolve'
+import terser from '@rollup/plugin-terser'
 import typescript from '@rollup/plugin-typescript'
+import { dts } from 'rollup-plugin-dts'
+
+import dotenv from 'dotenv'
+dotenv.config()
+
+const PROD = process.env.NODE_ENV === 'production'
```

Comment:
Let's change this to `!== development`, we don't have production set in actions yet